closed/src/java.base/share/classes/openj9/internal/security/RestrictedSecurity.java

```diff
@@ -593,17 +696,15 @@ private void initProviders() {
                 providerName = (pos < 0) ? providerName : providerName.substring(pos + 1, providerName.length());
                 // Provider without arguments and package names.
                 providersSimpleName.add(pNum - 1, providerName);
-
-                if (debug != null) {
-                    debug.println(
-                            "Loaded restricted security provider: " + providers.get(pNum - 1)
-                                    + " with simple name: " + providerName);
-                }
             }
 
             if (providers.isEmpty()) {
                 printStackTraceAndExit(
-                        "Restricted security mode provider list empty, or no such restricted security policy in java.security file.");
+                        "No providers were provided as part of the Restricted Security profile.");
```

Comment:
How about "No providers are specified as part of ..."

Reply:
Changed to your suggestion.

Reply:
"are" instead of "were" is better.

Reply:
Fixed.